input_cloud: /pointcloud_merger_node/merged_points
# CONDITIONAL_EUCLIDIAN_CLUSTERING = 0
# EUCLIDIAN_CLUSTER_EXTRACTION = 1
clustering_method: 1

downsample:
  enable: false

euclidean_clustering:
  cluster_tolerance: 1.0

conditional_euclidean_clustering:
  radius_search: 0.1
  cluster_tolerance: 500

min_cluster_size: 10
max_cluster_size: 300
inflation_size : 0.8